cms/static/js/spec/views/paging_spec.js

```diff
@@ -380,193 +384,5 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                     });
                 });
             });
-
-            describe("PagingFooter", function () {
```

Comment:
@andy-armstrong - In order for the assets view to render a header, footer, and paging view, it must include each one of those subviews. It is up to the assets spec to verify for itself that all three components have been composed appropriately. This `paging_spec.js` just happened to test three separate components in the same file. Including a `describe` for `PagingFooter` spec in this file will not solve that problem; even if it were to be added back, these tests would pass regardless of the design of the assets view.